lib/fhir_models/bootstrap/model.rb

```diff
@@ -6,7 +6,11 @@
 module FHIR
   class Model
     extend FHIR::Deprecate
+
+    attr_reader :source_text, :source_hash
```

Comment:
Do we really need to keep the source_hash?  We are already increasing the memory footprint of a model by >2x by keeping the source text of the text it was hydrated for.  If we store source_hash also, plus for each nested model in bundles, this update would increase the memory footprint of a FHIR resource instance by > 3x?

Reply:
Yes, we need it because many resources are instantiated from hashes.